recipes/lintegrate/LICENSE.txt

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
If the licence is in the tarball this can be deleted

Reply:
Thanks, I've deleted the license file now.